### PR TITLE
usb-modeswitch: update 2.6.0 -> 2.6.1

### PR DESCRIPTION
**Pull request by hoinmic**

### CODE REVIEW

File: meta-oe/recipes-support/usb-modeswitch/usb-modeswitch_2.6.1.bb [hunk 1]
```diff
@@ -5,16 +5,21 @@ LIC_FILES_CHKSUM = "file://COPYING;md5=091556bd6d0154cd4c2d17a1bfc7380a"
 DEPENDS = "libusb1"
 
 SRC_URI = "http://www.draisberghof.de/usb_modeswitch/${BP}.tar.bz2"
-SRC_URI[md5sum] = "be73dcc84025794081a1d4d4e5a75e4c"
-SRC_URI[sha256sum] = "c215236e6bada6e659fc195a31d611ea298a4bdb4d57a0d68c553b56585f8ba3"
+SRC_URI[md5sum] = "69d00d6ed7097b97d972a159d14cfba1"
```
Comment (kraj) [inline meta-oe/recipes-support/usb-modeswitch/usb-modeswitch_2.6.1.bb:8]: drop `md5sum` we don't need it any longer.

**Review by kraj — CHANGES_REQUESTED** [review]
Please add your `Signed-off-by` to the commit as well in v2

**Comment by hoinmic** [comment]
Modifications made. Thank you.

**Comment by kraj** [comment]
This is now in master via 13606a7ed3

**Pull request closed without merging**